solutions/USACO open16-closing-the-farm.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: usaco-644
+source: USACO Silver 2016 US Open
+title: Closing the Farm
+author: Jesse Choe
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_closing_silver_open16.html)
+
+<LanguageSection>
+<CPPSection>
+
+In this problem, we are asked to determine if each barn is connected after a certain barn is closed. To check this, we can run a depth-first search (DFS) algorithm
+on the last node to be closed, since each line asks whether the closure of the previous barn will allow connection between each of the unclosed nodes, making the last
```

Comment:
```suggestion
starting from the barn that will close last, since each line asks whether the closure of the previous barn will allow connection between each of the unclosed nodes, making the last
```